### DISCUSSION TITLE
Welcome to ObsidianPPV Discussions!

### DISCUSSION THREAD

**Discussion by lupickup** [tot0/ObsidianPPV#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

The GitHub default message is cool I guess. Down for people to discuss things related to Obsidian/PPV/LifeOS

**Comment by D7891** [comment]
Thank you for creating this PPV system in obsidian.

**Comment by svemagie** [comment]
Interesting! 
just moved from Obsidian to Notion for about 2 Months to evaluate the usefulness of PPV. Happy to see that you made PPV into Obsidian, so I'll probably move back to beloved plaintext obsidian! big thanks for your effort!

**Comment by Saorsa32** [comment]
How is the progress on adding the rest of the PPV vault coming? I can't wait!
